master/buildbot/test/unit/test_process_build.py

```diff
@@ -385,11 +385,11 @@ def claim(owner, access):
 
         b.startBuild(FakeBuildStatus(), None, self.workerforbuilder)
 
-        self.assert_(('startStep', (self.workerforbuilder.worker.conn,), {})
-                     not in step.method_calls)
+        self.assertNotIn(('startStep', (self.workerforbuilder.worker.conn,), {}),
+                     step.method_calls)
         self.assertEquals(claimCount[0], 1)
-        self.assert_(b.currentStep is None)
-        self.assert_(b._acquiringLock is not None)
+        self.assertTrue(b.currentStep is None)
```

Comment:
Is it 2.7 only? I think we agreed to not break 2.6 compatibility in master without special need on one of the last meetings.